cpp/benchmarks/utilities.cc

```diff
@@ -101,16 +101,16 @@ utilities::vis_params random_measurements(t_int size, const t_real max_w, const
   std::ifstream vis_file_str(vis_file);
 
   utilities::vis_params uv_data;
-  if (vis_file_str.good()) {
+  if (false) {
```

Comment:
This looks suspect, why would we have an `if (false)` statement?

Reply:
This is the part where the code is writing the visibilities to a file and re-using them, that was failing when multiple jobs tried to access the same file. It might be useful to keep the option there but I'll make it a bit more explicit.